### ISSUE TITLE
fputcsv() flagged when writing to stream

### ISSUE THREAD

**Issue by paulschreiber** [WordPress-Coding-Standards/WordPress-Coding-Standards#295]
I have some code that creates a CSV for download:

``` php
$output_stream = fopen( 'php://output', 'w' );

foreach ( $list as $row ) {
    fputcsv( $output_stream, $row );
}
```

PHPCS flags this as an error:

```
  88 | ERROR   | Filesystem writes are forbidden, you should not be using
     |         | fputcsv() (WordPress.VIP.FileSystemWritesDisallow)
```

I'm writing to a stream, not the filesystem.

**Comment by grappler** [comment]
There is no problem in the sniff. Perhaps the Wording should be improved. This could be labeled with `vip`

**Comment by paulschreiber** [comment]
Filed https://github.com/Automattic/VIP-Coding-Standards/issues/404.
